cmd/gen/workflows/flag.go

```diff
@@ -0,0 +1,26 @@
+package workflows
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagDir = "dir"
+)
+
+type flag struct {
+	Dir string
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&f.Dir, flagDir, "./.github/workflows", `Directory where the generated code should be located.`)
```

Comment:
I wonder if this will change? or if we can drop it similar to Makefile

Reply:
I dropped this flag.